index.bs

```diff
@@ -549,7 +554,8 @@ To support obtaining assertions via {{CredentialsContainer/get()|navigator.crede
 {{CredentialsContainer/create()|navigator.credentials.create()}} to request the creation of a new [=credential key pair=]
 and {{PublicKeyCredential}}, managed by an [=authenticator=].
 On success, the returned {{promise}} will be resolved with a {{PublicKeyCredential}} containing an
-{{AuthenticatorAttestationResponse}} object.
+{{AuthenticatorAttestationResponse}} object. This method can be <dfn id="concept-webauthn-terminated">terminated</dfn> 
+with flag [=AbortSignal/aborted flag=], which is unset unless otherwise specificed.
```

Comment:
This seems wrong. How many reasons are there to be terminated? Just aborting?

Reply:
There are two reasons why an create credential and get credential should be terminated: 1) JS code intentionally terminates the operation to provide better experience, 2) a document unloads and asks all authentication operations to be terminated. The second one is the one I am still not sure what to do with. Preferably there should be only one abort eventListener for the browser to know whether it should start aborting the request. Weird things like page unload should just set the abort flag.

Reply:
I wonder if we have a consistent story for unload. I suspect a lot of promises might end up forever pending whenever that happens.